Dockerfile

```diff
@@ -1,13 +1,45 @@
+FROM docker.io/golang:alpine as builder
+
+WORKDIR /src
+
+ENV CGO_ENABLED=0
+
+RUN apk add git 
+RUN git clone https://github.com/yggdrasil-network/yggdrasil-go.git .
+RUN ./build && go build -o /src/genkeys cmd/genkeys/main.go
+
+
 FROM golang:1.16-alpine
 
 WORKDIR /grid_proxy_server
 
+RUN apk --update add redis 
+
 COPY go.mod go.sum ./
 RUN go mod download
 
 COPY . .
 RUN go build -o server cmds/proxy_server/main.go
 
+RUN wget https://github.com/threefoldtech/zinit/releases/download/v0.1/zinit -O /sbin/zinit \
+    && chmod +x /sbin/zinit
+
+# TODO: copy msgbus from release, currently locally
+COPY msgbus /usr/bin/msgbus
```

Comment:
suggestion:
clone `rmb-go` repo in the builder and build it.


_the current stable branch in `rmb-go` repo is `testing`_